src/exporter.js

```diff
@@ -124,6 +125,10 @@ export default class {
   }
 }
 
+export const SPACE_REPLACER = String.fromCharCode(160);
```

Comment:
yep. But then anki displays two tags `vocab` and `check_with_tutor`. I check it before. It's variant, and I agree to replace `\u160` with `_` but it looks ugly